PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.webapp/src/com/microsoft/azuretools/webapp/ui/AppServiceCreateDialog.java

```diff
@@ -1093,6 +1099,8 @@ protected void okPressed() {
                         Display.getDefault().asyncExec(() -> AppServiceCreateDialog.super.cancelPressed());
                     }
                     try {
+                        TelemetryUtil.sendTelemetryOpStart("create web app", properties);
+                        long timeS = System.currentTimeMillis();
```

Comment:
time, timeS is not readable.

Reply:
timeS means timeStart, or just use timeStart